### DISCUSSION TITLE
[BUG] can't pull images from registry.cn-hongkong.aliyuncs.com

### DISCUSSION THREAD

**Discussion by zhangzhuoBYS** [alibaba/GraphScope#2485] [Q&A] [answered]
**Describe the bug**
can't pull images from registry.cn-hongkong.aliyuncs.com！
when use helm to install the cluster flollowing by the Official manual
, `ImagePullBackOff` raised.
![图片](https://user-images.githubusercontent.com/40155703/223000973-db47b3f5-0e07-4e87-9263-2eff065aefc8.png)

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
 - GraphScope version: [e.g., v0.1, v0.2.2, master]
 - OS: [e.g. MacOS, Linux]
 - Version [e.g. 10.15]
 - Kubernetes Version [e.g., 1.19]

**Additional context**

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! And a maintainer will get back to you shortly!
Please feel free to contact us on [DingTalk](https://h5.dingtalk.com/circle/healthCheckin.html?dtaction=os&corpId=dingb4b76e0fa9ccd8d516fd0f0acdf19f55&68e71190=0fe34ec0&cbdbhh=qwertyuiop), WeChat account(graphscope) or [Slack](http://slack.graphscope.io/). We are happy to answer your questions responsively.

**Comment by sighingnow** [comment, marked as answer]
From the error message you can see that containerd was complaining that the disk is full is no space left, thus failed to pull the docker image.

It is not a bug of graphscope. You could try to cleanup your disk to reserve spaces for images and try gain.

**Comment by sighingnow** [comment]
Closing as moving to discussion.
